### PR TITLE
neutron: Replace /etc/sysconfig/neutron usage with links

### PR DESCRIPTION
**Pull request by toabctl**
/etc/sysconfig/neutron is no longer supported. Instead use symlinks
to enable/disable specific configs for the neutron-server.

### CODE REVIEW

File: chef/cookbooks/neutron/recipes/server.rb [hunk 1]
```diff
@@ -242,17 +251,12 @@
...
-             for i in $NEUTRON_PLUGIN_CONF; do \
-               CONF_ARGS="$CONF_ARGS --config-file $i"; \
-             done; \
-             neutron-db-manage --config-file /etc/neutron/neutron.conf $CONF_ARGS upgrade head'
   when "debian"
     command 'source /etc/default/neutron-server; \
              neutron-db-manage --config-file /etc/neutron/neutron.conf --config-file $NEUTRON_PLUGIN_CONFIG upgrade head'
   else
-    command "neutron-db-manage --config-file /etc/neutron/neutron.conf upgrade head"
+    # FIXME(toabctl): Does neutron-db-manage need the ml2 plugin configs???
```
Comment (dirkmueller) [inline chef/cookbooks/neutron/recipes/server.rb]: I think it does.. its the one that initializes the core plugin specific database, so it needs to know about the core plugin configuration.

Comment (toabctl) [inline chef/cookbooks/neutron/recipes/server.rb]: Then I wonder how we want to handle this in the package case. There we have in the service file ```ExecStartPre=-/usr/bin/neutron-db-manage upgrade head```

Comment (aplanas) [inline chef/cookbooks/neutron/recipes/server.rb]: @toabctl For me makes sense

Comment (toabctl) [inline chef/cookbooks/neutron/recipes/server.rb]: @aplanas what makes sense? How would you fix it in the service file?

Comment (aplanas) [inline chef/cookbooks/neutron/recipes/server.rb]: @toabctl uhm I was understanding that your suggestion was to add `ExecStartPre` in each plugin service, and not only in openstack-neutron.service one. Did I understand wrong?

Comment (toabctl) [inline chef/cookbooks/neutron/recipes/server.rb]: @dirkmueller done

**Pull request merged**